cmake/BuildUnicyclePlanner.cmake

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2018  iCub Facility, Istituto Italiano di Tecnologia
+# Authors: Stefano Dafarra <stefano.dafarra@iit.it>
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(iDynTree 0.9.2 QUIET)
+
+set(UnicyclePlanner_DEPENDS "")
+list(APPEND UnicyclePlanner_DEPENDS iDynTree)
```

Comment:
If you have just a limited amount of dependency, you can just list them after `DEPENDS` (but this is ok as well).

Reply:
I did it mechanically.